ui/page/settings/settings_page.go

```diff
@@ -624,21 +621,25 @@ func (pg *SettingPage) updateSettingOptions() {
 }
 
 func (pg *SettingPage) updatePrivacySettings() {
-	pg.setInitialSwitchStatus(pg.privacyActive, pg.WL.AssetsManager.IsPrivacyModeOn())
-	if !pg.WL.AssetsManager.IsPrivacyModeOn() {
-		pg.setInitialSwitchStatus(pg.transactionNotification, pg.WL.AssetsManager.IsTransactionNotificationsOn())
-		pg.setInitialSwitchStatus(pg.governanceAPI, pg.WL.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.GovernanceHTTPAPI))
-		pg.setInitialSwitchStatus(pg.exchangeAPI, pg.WL.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.ExchangeHTTPAPI))
-		pg.setInitialSwitchStatus(pg.feeRateAPI, pg.WL.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.FeeRateHTTPAPI))
-		pg.setInitialSwitchStatus(pg.vspAPI, pg.WL.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.VspAPI))
-	} else {
+	pg.setInitialSwitchStatus(pg.privacyActive, pg.AssetsManager.IsPrivacyModeOn())
+	if !pg.AssetsManager.IsPrivacyModeOn() {
+		pg.setInitialSwitchStatus(pg.transactionNotification, pg.AssetsManager.IsTransactionNotificationsOn())
+		pg.setInitialSwitchStatus(pg.governanceAPI, pg.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.GovernanceHTTPAPI))
+		pg.setInitialSwitchStatus(pg.exchangeAPI, pg.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.ExchangeHTTPAPI))
+		pg.setInitialSwitchStatus(pg.feeRateAPI, pg.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.FeeRateHTTPAPI))
+		pg.setInitialSwitchStatus(pg.vspAPI, pg.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.VspAPI))
+	} /*else {
+		// TODO: Re "Clear all the peers saved if the privacy mode is on". Does
+		// this mean clear all peers for all wallets? The privacy mode isn't on
+		// for just the selected wallet afterall.
+
 		if pg.WL.SelectedWallet != nil {
 			go func() {
 				// Clear all the peers saved if the privacy mode is on.
 				pg.WL.SelectedWallet.Wallet.SetStringConfigValueForKey(sharedW.SpvPersistentPeerAddressesConfigKey, "")
 			}()
 		}
```

Comment:
This is temporarily commented out, but I want to remove completely. I don't think it is proper to clear previously saved peer addresses because the user enabled privacy mode. If privacy mode is later turned off, the user has to re-enter peer addresses? I think this should be addressed same way it is done for currency exchange configuration. Keep the value but don't use it if privacy mode is on. Thoughts?